charts/operator/templates/clusterrole.yaml

```diff
@@ -295,6 +296,53 @@ metadata:
   labels:
 {{ include "redpanda-operator.labels" . | indent 4 }}
 rules:
+- apiGroups:
+    - rbac.authorization.k8s.io
+  resources:
+    - clusterrolebindings
+    - clusterroles
+  verbs:
+    - create
+    - get
+    - delete
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - configmaps
+    - endpoints
+    - events
+    - limitranges
+    - persistentvolumeclaims
+    - pods
+    - pods/log
+    - replicationcontrollers
+    - resourcequotas
+    - serviceaccounts
+    - services
+  verbs:
+    - get
+    - list
```

Comment:
I'm against this change as namespaced operator will not be a thing.

Reply:
Let's Discuss this PR then, The issue needs to be resolved.

Reply:
Best alternative i found after talking to @joejulian was to just gate this option. So added a new values that will deploy this if user really wants to enable rpk-bundle clusterroles to redpandas.